wurst/systems/units/hostiles/LesserHydra.wurst

```diff
@@ -95,9 +94,8 @@ public class LesserHydra extends Hostile
 
     override function onDeath()
         deathPos = getPos()
-        removeUnit()
```

Comment:
We definitely want to remove the unit here

Reply:
Are you sure about that? The ondestroy function removes the unit later